src/main/scala/com/johnsnowlabs/util/Build.scala

```diff
@@ -0,0 +1,16 @@
+package com.johnsnowlabs.util
+
+import com.johnsnowlabs.nlp.util.io.ResourceHelper
+
+object Build {
+  val version: String = {
+    val objPackage = ResourceHelper.getClass.getPackage
```

Comment:
This triggers the creation of a Spark session when the ResourceHelper object is built. I this really intended behavior?